addons/survey/views/survey_question_views.xml

```diff
@@ -29,42 +28,50 @@
                                 <div attrs="{'invisible': [('question_type', '!=', 'date')]}">
-                                    <p class="o_datetime">YYYY-MM-DD
+                                    <i>When is Christmas?</i><br/>
+                                    <p class="o_datetime" style="border:none">YYYY-MM-DD
                                         <i class="fa fa-calendar fa-2x" role="img" aria-label="Calendar" title="Calendar"/>
                                     </p>
                                 </div>
                                 <!-- Date and Time -->
                                 <div attrs="{'invisible': [('question_type', '!=', 'datetime')]}">
-                                    <p class="o_datetime">YYYY-MM-DD hh:mm:ss
+                                    <i>When does ... start?</i><br/>
+                                    <p class="o_datetime" style="border:none" >YYYY-MM-DD hh:mm:ss
                                         <i class="fa fa-calendar fa-2x" role="img" aria-label="Calendar" title="Calendar"/>
                                     </p>
                                 </div>
                                 <!-- Multiple choice: only one answer -->
                                 <div attrs="{'invisible': [('question_type', '!=', 'simple_choice')]}" role="img" aria-label="Multiple choice with one answer" title="Multiple choice with one answer">
-                                    <div class="row"><i class="fa fa-circle-o  fa-lg"/> answer</div>
-                                    <div class="row"><i class="fa fa-dot-circle-o fa-lg"/> answer</div>
-                                    <div class="row"><i class="fa fa-circle-o  fa-lg"/> answer</div>
+                                    <i>Which is yellow?</i><br/>
+                                    <div class="row"><i class="fa fa-circle-o  fa-lg"/> &#160;&#160;answer</div>
+                                    <div class="row"><i class="fa fa-dot-circle-o fa-lg"/> &#160;&#160;answer</div>
+                                    <div class="row"><i class="fa fa-circle-o  fa-lg"/> &#160;&#160;answer</div>
                                 </div>
                                 <!-- Multiple choice: multiple answers allowed -->
                                 <div attrs="{'invisible': [('question_type', '!=', 'multiple_choice')]}" role="img" aria-label="Multiple choice with multiple answers" title="Multiple choice with multiple answers">
-                                    <div class="row"><i class="fa fa-square-o fa-lg"/> answer</div>
-                                    <div class="row"><i class="fa fa-check-square-o fa-lg"/> answer</div>
-                                    <div class="row"><i class="fa fa-square-o fa-lg"/> answer</div>
+                                    <i>Which are yellow?</i><br/>
+                                    <div class="row"><i class="fa fa-square-o fa-lg"/> &#160;&#160;answer</div>
```

Comment:
I can see you use "&#160;" for spacing (I'm guessing).

Please use spacing classes instead (e.g: "mr-1", see bootstrap doc for more details).
Apply for other occurrences of "&#160;" too.